.cloudbuild/deploy.yaml

```diff
@@ -0,0 +1,12 @@
+steps:
+  - name: node
+    entrypoint: npm
+    args: ['ci']
```

Comment:
Why do we do some steps here (`ci` and `cloud-secrets`) but do other steps in the firebase predeploy `build` and `algolia`? Is it because `ci` and `cloud-secrets` aren't needed if someone is deploying from their local machine?

Reply:
Yea, the cloud secrets only needs (can be) run when in a cloud build environment.